src/app/components/course/CourseDetails/CourseDetailsTopBanner/CourseDetailsTopBanner.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import React from 'react';
+import styles from '../../CourseDetails/courseDetails.module.css';
+import Typography, { TypographyVariant } from '../../../typography/Typography';
+import classNames from 'classnames';
+import Image from 'next/image';
+import Logo from '@/app/images/Logo-navbar.svg';
+import ActionButton from '../../../buttons/ActionButton';
+import { BUTTON_STYLE } from '@/app/utilities/constants';
+import { useCourseDetailsContext } from '@/app/utilities/course/CourseDetailsProvider';
+import CourseDetailsMiddleBanner from '../CourseDetailsMiddleBanner/CourseDetailsMiddleBanner';
+
+const CourseDetailsTopBanner: React.FC = () => {
+  const { data } = useCourseDetailsContext();
+
+  return (
+    <div className={classNames('row', styles.topBannerMainContainer)}>
+      <div className={classNames('col-md-7')}>
```

Comment:
Not the most important, but if possible, when there's only one class, avoid using `classNames`.